src/components/UI/Modal/pages/LoginModal.tsx

```diff
@@ -131,6 +133,24 @@ function getAutoConnectBody(data: ILoginPage): JSX.Element {
 
 function getVerifyingBody(data: ILoginPage): JSX.Element {
 	if (data.connectionType === LoginConnectionType.Sso) {
+		if (data.ssoVerification) {
+			return (
+				<div>
+					<div className={css.codeContainer}>
+						<div className={css.codeBody}>Verification Code</div>
+						<div className={css.codeBody}>
+							<div>{data.ssoVerification.code}</div>
+							<div>
+								<CopyButton copyText={data.ssoVerification.code} />
```

Comment:
Looking in the image in the PR description, the code and the button are not in the center